crates/sui-rosetta/src/types.rs

```diff
@@ -775,7 +775,7 @@ impl InternalOperation {
                 amounts,
             }),
         };
-        TransactionData::new(
+        TransactionData::new_with_dummy_gas_price(
```

Comment:
Should we add a new `gas_price` field to `ConstructionMetadata` @patrickkuo ?

Reply:
This is good for now, I will check with CB to see if we need to be able to set price or not